test/extended/oauth/requestheaders.go

```diff
@@ -288,6 +291,57 @@ func gatherPostMortem(oc *exutil.CLI) {
 		}
 		e2e.Logf("log from %s in %s: %s", pod.Name, oauthServerNamespace, logs)
 	}
+
+	operator, err := oc.AdminOperatorClient().
```

Comment:
the clusteroperator state is gathered in an aggregated form above, I don't think this is necessary

Reply:
The inertia logic only drops the "True" from Degraded state but AFAIK the messages are kept and aggregated. That being said, I think I wouldn't be against logging this.

However, too much information might also be confusing for the reader, so I would rather be a little careful about what we're posting here.

Btw - the location of these logs is wrong. It should be:
```
clusteroperator state
operator state
deployment
configmap
pods
```
==> move this to the second place

Reply:
Yes, I know that the message stays. This is why you came up with the working hypothesis, that something is in the bushes. I wanted to make it more explicit who or what is in a bad shape. The messages are very cryptic, if torn out of context.